src/main/java/com/iota/iri/conf/MilestoneConfig.java

```diff
@@ -6,12 +6,16 @@
 public interface MilestoneConfig extends Config {
 
     /**
-     * @return Descriptions#COORDINATOR
+     * Default Value: {@value BaseIotaConfig.Defaults#COORDINATOR_ADDRESS}
+     *
+     * @return {@value MilestoneConfig.Descriptions#COORDINATOR}
      */
     String getCoordinator();
 
     /**
-     * @return {@value Descriptions#DONT_VALIDATE_TESTNET_MILESTONE_SIG}
+     * Default Value: <tt>false</tt>
```

Comment:
why break the pattern of `Default Value: {@value ...`?

Reply:
has a default: `DONT_VALIDATE_MILESTONE_SIG` - which doesn't match the description name :\